api/http/handler/endpoints/endpoint_update.go

```diff
@@ -211,5 +211,41 @@ func (handler *Handler) endpointUpdate(w http.ResponseWriter, r *http.Request) *
 		}
 	}
 
+	if endpoint.Type == portainer.EdgeAgentEnvironment {
```

Comment:
Should we always trigger the relation update process here? What if I only updated the name of the endpoint?

Reply:
do you mean checking if groupID or tags were changed?